.github/workflows/build.yaml

```diff
@@ -88,13 +87,11 @@ jobs:
 
       - name: Install release from Test PyPI
         run: >
-          poetry add
-          --source test-pypi
-          packse==${{ needs.build-and-publish.outputs.build-version }}
+          uv pip install --system --extra--index-url https://test.pypi.org/simple/ packse==${{ needs.build-and-publish.outputs.build-version }}
```

Comment:
One too many dashes in `--extra--index-url`

Reply:
Thanks!